.github/workflows/pull_request.yml

```diff
@@ -38,3 +38,18 @@ jobs:
         env:
           IMG_RESULT: cache
         run: make buildx.image
+
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'openebs/linux-utils:ci'
```

Comment:
:+1:

Reply:
@prakhargurunani - as trivy is executed as a new job, it seems to launch a new VM, pull the older image and run the tests. Can you try to push the step of running the check after "Build Image" step. So the change would look something like:

Reply:
```
jobs:
  linux-utils:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Set up Docker Buildx
        id: buildx
        uses: crazy-max/ghaction-docker-buildx@v1
        with:
          buildx-version: latest
      - name: Build Image
        env:
          IMG_RESULT: cache
        run: make buildx.image
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: 'openebs/linux-utils:ci'
```

Reply:
@kmova Done.